### PR TITLE
Build time speedups

### PR DESCRIPTION
**Pull request by studywolf**
ready to merge! 

This drops the buildtime by a factor of 10-20. 
Just used numpy instead of creating theano functions for the encoders and random numbers in ensemble.py.

### CODE REVIEW

File: nengo/scripts/scrap.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+from .. import nef_theano as nef
```
Comment (jaberg) [inline nengo/scripts/scrap.py:1]: did you mean to include this file?

**Comment by jaberg** [comment]
Also, in the spirit of reducing up build times, I found it more natural to use numpy.linalg.lstsq() instead of the various dots and eigh stuff in ensemble_origin. It's probably faster too.  Does it work in general?

```
A = neurons.accumulate(...)
decoders = lstsq(A, target_values)
```

**Comment by studywolf** [comment]
gaaahhhh wtf. the only thing i wanted to include was the very last commit, the one from today. that one only changes ensemble.py ... sigh. i'll just delete this and try again.

**Comment by studywolf** [comment]
good question! thought @tcstewar ?

**Pull request closed without merging**